Marsiple/App/Todo Form/TodoFormViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  TodoFormViewController.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 31/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class TodoFormViewController: UIViewController {
+    // NOTE: id is set to -1 so we can detect if we're creating new or changing existing todo (to detect wich initializer is used)
+    private var todo = Todo(id: -1, title: "", completed: false, userId: 1)
+    private let todoFormView = TodoFormView.autolayoutView()
+    private lazy var completedButton: UIButton = {
```

Comment:
We could set-up the button as so: 
```
private lazy var completedButton: UIButton = {
        let button = UIButton()
        button.setImage(#imageLiteral(resourceName: "checkbox-unselected"), for: .normal)
        button.setImage(#imageLiteral(resourceName: "checkbox-selected"), for: .selected)
        button.isSelected = todo.completed
        button.addTarget(self, action: #selector(completedButtonTapped), for: .touchDown)
        button.snp.makeConstraints { $0.width.height.equalTo(30) }
        return button
}()
```

The interesting parts are the following lines:
```
button.setImage(#imageLiteral(resourceName: "checkbox-unselected"), for: .normal)
button.setImage(#imageLiteral(resourceName: "checkbox-selected"), for: .selected)
```

You can set an properties based on the `UIButton`'s states, like we did above. So, for the `normal` state we want to use the `unselected image`, but for the `selected` state, we want to use the `selected image`. This way, we can just mutate the `isSelected` property and get the button to update automatically.

**Note:**
This works for other properties on the `UIButton`, such as the `title`, `titleColor` and so on.